library/swift/Headers.swift

```diff
@@ -4,45 +4,58 @@ import Foundation
 /// To instantiate new instances, see `{Request|Response}HeadersBuilder`.
 @objcMembers
 public class Headers: NSObject {
-  let headers: [String: [String]]
+  let container: HeadersContainer
 
   /// Get the value for the provided header name.
   ///
-  /// - parameter name: Header name for which to get the current value.
+  /// - parameter name: The case-insensitive header name for which to
+  ///                   get the current value.
+  ///
   ///
   /// - returns: The current headers specified for the provided name.
   public func value(forName name: String) -> [String]? {
-    return self.headers[name]
+    return self.container.value(forName: name)
   }
 
   /// Accessor for all underlying headers as a map.
   ///
   /// - returns: The underlying headers.
   public func allHeaders() -> [String: [String]] {
```

Comment:
Consider renaming this function to `caseSensitiveHeaders()` to raise awareness to the fact that users will likely need to perform case-insensitive lookups if they want to find values for given keys.

Reply:
updated the name